packages/embark-events/README.md

```diff
@@ -0,0 +1,7 @@
+embark-events
```

Comment:
what's is the reasoning for creating a package for this? the events are integral part of thee core api, IMO they shouldn't another package, we have a bit too many packages already.

Reply:
The reasoning is so that we can use a private instance embark events inside of another package `embark-rpc-manager`, so as not to pollute the global events (which could incorrectly allow other developers to use events they're not meant to be using - and potentially cause race conditions).